workflow-templates/shellcheck.yml

```diff
@@ -0,0 +1,65 @@
+# ShellCheck is a linter for your shell scripts:
+#   https://www.shellcheck.net/
+# This workflow runs it on PRs and pushes to $default-branch
+
+name: ShellCheck
+
+on:
+  pull_request:
+  push:
+    branches:
+    - $default-branch
+
+permissions:
+  contents: read
+
+jobs:
+
+  shellcheck:
+    name: Run ShellCheck
+    runs-on: ubuntu-latest
+
+    steps:
+
+    - name: Check out branch
+      uses: actions/checkout@v3
+
+    # Run ShellCheck using a predefine action:
+    # https://github.com/marketplace/actions/shellcheck
+    - name: Run ShellCheck
+      uses: ludeeus/action-shellcheck@master
```

Comment:
discussion from https://github.com/openedx/.github/pull/60 (@timmc-edx)

![image](https://user-images.githubusercontent.com/3628148/221642151-ee5b4926-91ae-4d13-95a0-3226125a3584.png)

Reply:
@timmc-edx Agreed. I was a little concerned about the work required to bump this version across repositories (in case of a security issue in the action, for example), but if we go with @feanil 's suggestion above and use a shared workflow, then the action version could be controlled from one place (if we even continue to use the action, that is).

Reply:
This is done.